chapters/Chapter 2 — Dressara.md

```diff
@@ -0,0 +1,94 @@
+The women of Dressara often wore dresses, which Leanne both admired and found a shame. Leanne enjoyed wearing dresses, but found women in suits much more handsome.
+
+Delphine was nothing if not handsome. She was tall, and did her best to look so: her fine Elisian suit lengthened her torso, and she had paired it, rather than with pants, with a skirt just long enough to wave gently in the light breeze the Dressarans were so fond of.
+
+Dressara was a large moon. It was somewhat pleasant, if you enjoyed their sort of pomp, unlike the dour planet Seven around which it held a lazy orbit. Like anywhere else, it was rather difficult to land on illegally, particularly when it was in the process of adjusting its orbit. 
```

Comment:
> around which it held a lazy orbit

"around which it held its lazy orbit

> it was rather difficult to land on illegally, particularly when it was in the process of adjusting its orbit

I don't know how much orbital adjustment would really mess things up. I'd like to mention the orbit adjustment, though. 

I'd also like to mention the two layers of defense the planetary bodies have.